### PR TITLE
personalize caller experience with avatar and name

### PR DESCRIPTION
**Pull request by julieannenoying**
When you are waiting for the person who shared the call link to join, it would be nice to have a more personalized interface. 

I copied the snippet from the "files drop", but lack the knowledge to make the Javascript work...

Please help @MorrisJobke @Ivansss :)

### CODE REVIEW

**Comment by MorrisJobke** [comment]
@rullzer Could you help here, please? Thanks 

Sidenote: @rullzer added the avatars for disabled videos. But basically it is a `$('#displayavatar').avatar(username, size);` but I'm not 100% sure where to add regarding initialisation.

**Comment by Ivansss** [comment]
What about showing the sharing link?

screen shot 2016-11-28 at 10 35 56

We implemented different waiting messages for different types of rooms in this PR #144 

Normal users will see the image above when they are waiting in a public room, so I think we could show the same view to a non-registered user to make it consistent :)

What do you think @julieannenoying @MorrisJobke @jancborchardt ?

**Comment by jancborchardt** [comment]
@Ivansss we also thought about that, but say when you have a 1o1 call and you wait for the other person to join. It would be a strange context to give the guest the link to share and display this generic message of »Waiting for people to join the call«. It’s better to give feedback which person is missing, and we know at least one because that person originally created the room. :)

And about the link – as soon as one other person joins, the link input etc won’t be visible, so we need a different solution for this anyway if we want to show the link to guests too.

**Comment by nickvergessen** [comment]
For me this is not displayed, instead I see my own video full window width?
Anything I need to do before seeing this?

**Comment by jancborchardt** [comment]
@nickvergessen 
1. Create a public room, but don’t join it
2. Open that link in an incognito window
3. Content should be empty, and in the bottom right there’s your video

File: js/app.js [hunk 1]
```diff
@@ -246,14 +246,65 @@
...
 					success: function() {
 						roomChannel.trigger('active', roomId);
 					}
 				});
+			} else {
+				$.ajax({
+					url: OC.generateUrl('/apps/spreed/api/room/') + roomId,
+					type: 'GET',
+					success: function(data) {
+						console.log("data", data);
```
Comment (nickvergessen) [inline js/app.js]: kill?

**Comment by jancborchardt** [comment]
@Ivansss cool you work on it! Is this ready for review? :)

**Comment by Ivansss** [comment]
@jancborchardt yes, it could be reviewed :)

**Comment by jancborchardt** [comment]
Looks good to me :+1: 

The only small issue is that the avatar placeholder color doesn’t match:
capture du 2016-12-02 14-08-58
Any idea why this happens? cc @rullzer @Kondou-ger

**Comment by jancborchardt** [comment]
Great collaboration @Ivansss @julieannenoying! :))

**Pull request merged**